.github/workflows/python-ci.yml

```
@@ -12,7 +12,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

gotta get the other one too!
```suggestion
    runs-on: ubuntu-latest
```